infrastructure/terraform/critical/cognito.tf

```diff
@@ -23,6 +23,24 @@ resource "aws_cognito_user_pool_domain" "domain" {
   user_pool_id    = "${aws_cognito_user_pool.pool.id}"
 }
 
+resource "aws_cognito_resource_server" "stacks_api" {
+  identifier = "https://api.wellcomecollection.org/stacks/v1"
+  name       = "Stacks API V1"
+
+  scope = [
+    {
+      scope_name        = "requests"
+      scope_description = "Read and write requests"
+    },
+    {
+      scope_name        = "items"
+      scope_description = "Read the status of items"
```

Comment:
FWIW, I'd bake the permissions into the scope names, so if I just see the scope name, it's clear at a glance what I get. i.e.

```hcl
{
  scope_name = "requests_readwrite",
},
{
  scope_name = "items_readonly",
}
```